src/Makefile

```diff
@@ -30,8 +30,8 @@ P11KIT_CFLAGS = $(shell $(PKG_CONFIG) --cflags p11-kit-1)
 P11KIT_LIBS   = $(shell $(PKG_CONFIG) --libs   p11-kit-1)
 TASN1_CFLAGS = $(shell $(PKG_CONFIG) --cflags libtasn1)
 TASN1_LIBS   = $(shell $(PKG_CONFIG) --libs   libtasn1)
-#HOWTOUSE# UNBOUND_FLAGS = $(shell $(PKG_CONFIG) --cflags libunbound)
-#HOWTOUSE# UNBOUND_LIBS  = $(shell $(PKG_CONFIG) --libs   libunbound)
+UNBOUND_CFLAGS = 
+UNBOUND_LIBS   = -lunbound
```

Comment:
I have no quarrel with this.  It may be that NixOS shows a special quality here -- where the link to libunbound inside GnuTLS may have saved Debian builds from not finding the library, this is unthinkable for NixOS' separation of library includes, so it ran into a problem.  Sweet (if that is the case).

Due to the misunderstanding in the other commit I will reject this pull request (or you might pull it back) and I will make a separate commit mentioning your contribution in the commit log.

Thanks, I'm happy that the core problem was so simple -- that's how it ought to have been.

Please understand why there's just a Makeful -- I've experimented with autoconfig; it made TLS Pool less stable than ever, especially building on other platforms.  And it's not as if there's a conceptual framework that helped me establish certainty that configurations were good at any point; I was patching without understanding the big picture, which was really frustrating.  At that point I've decided to return to plain Makefiles, which meant everything was stable again.

The quest for a good portability instrument remains.  You've mentioned CMake, and indeed that is a tool that we're going to look into.  It's just one of *many* things on the TODO lists for our projects :'-(